src/common.js

```diff
@@ -187,3 +195,7 @@ export function resolveImportMap (importMap, resolvedOrPlain, parentUrl) {
   }
   return applyPackages(resolvedOrPlain, importMap.imports) || resolvedOrPlain.indexOf(':') !== -1 && resolvedOrPlain;
 }
+
+export function errMsg(errCode, msg) {
+  return "SystemJS #" + errCode + " (" + "https://git.io/JvFET#" + errCode + "): " + (msg || "");
```

Comment:
Error conventions usually place the specific error message first, followed by the more general context last, so I'd suggest moving the link to the end. Also we might be able to cut a few more bytes off the `s.js` build here:

```suggestion
  if (DEV)
    return (msg || "") + " (SystemJS Error#" + errCode + " " + "https://git.io/JvFET#" + errCode + ")";
  else
    return (msg || "") + " (SystemJS https://git.io/JvFET#" + errCode + ")";
```